website/docs/cloud-docs/no-code-provisioning/module-design.mdx

```diff
@@ -46,6 +47,16 @@ For example, you could build modules that satisfy the following well-scoped use
 - Deploying all resources needed for a three-tier web application
 - Deploying a database with constraints on resource allocation and deployment region
 
+### Adjust for New Module Versions
```

Comment:
I used "Adjust for new" here because it's something they may do periodically over time, as new versions of a module get published. When a new version becomes available, the admins managing no-code status for a module will need to decide if they want to change the version pin and update the variable's input options.

Is there a better word to use here to describe this periodic maintenance?

Reply:
Thank you for the context here! I think the word "update" might be a better fit here if that is what admins will most commonly be doing? How does this feel?
```suggestion
### Updating a Module's version
```